data/json/items/book/tailor.json

```
@@ -144,6 +147,10 @@
     "looks_like": "textbook_firstaid",
     "color": "blue",
     "skill": "tailor",
+    "proficiencies": [
+      { "proficiency": "prof_closures", "time_factor": 0.1, "fail_factor": 0.25 },
+      { "proficiency": "prof_elastics", "time_factor": 0.1, "fail_factor": 0.25 }
```

```suggestion
      { "proficiency": "prof_closures", "time_factor": 0.1, "fail_factor": 0.3 },
      { "proficiency": "prof_elastics", "time_factor": 0.1, "fail_factor": 0.3 }
```
But also increase the time to 55 minutes. The fail_factor might be increased further depending on how many recipes it has.